lib/rollbar/util/obfuscator.rb

```diff
@@ -0,0 +1,15 @@
+module Rollbar
+  module Util
+    module Obfuscator
+      require 'ipaddr'
+      require 'digest'
+
+      def self.obfuscate_ip(ip_string, secret)
+        ip_int32 = IPAddr.new(ip_string, Socket::AF_INET).to_i
```

Comment:
I'd prefer call this method always and just do `return ip_string unless Rollbar.configuration.user_ip_obfuscator_secret`.

Probably this method should use `rescue` at some point and probably return original ip address string.